avro/profile.json

```diff
@@ -0,0 +1,160 @@
+{
+  "name":"profile",
+  "type":"record",
+  "namespace": "dto",
+  "doc":"Schema for cosmos api container 'profiles'",
+  "fields":[
+    {
+      "name": "id",
+      "type": "string"
+    },
+    {
+      "name":"version",
+      "type":"int",
+      "default":0
+    },
+    {
+      "name":"acceptedTosVersion",
+      "type": "int",
+      "default": "0"
+    },
+    {
+      "name":"blockedInboxOrChannels",
+      "type":{
+        "type": "map",
+        "values":{
+          "type": "array",
+          "items": {
+            "type":"enum",
+            "name":"BlockedInboxOrChannelEnum",
+            "symbols":[
+              "EMAIL",
+              "WEBHOOK",
+              "INBOX"
+            ]
+          }
+        }
+      },
+      "default" : {}
+    },
+    {
+      "name": "email",
+      "type": "string"
+    },
+    {
+      "name":"fiscalCode",
+      "type": "string"
+    },
+    {
+      "name": "isEmailEnabled",
+      "type": "boolean"
+    },
+    {
+      "name": "isEmailValidated",
+      "type": "boolean"
+    },
+    {
+      "name": "isInboxEnabled",
+      "type": "boolean"
+    },
+    {
+      "name": "isWebhookEnabled",
+      "type": "boolean"
+    },
+    {
+      "name": "isTestProfile",
+      "type": ["null","boolean"],
+      "default": null
+    },
+    {
+      "name":"lastAppVersion",
+      "type": ["null","string"],
+      "default":null
```

Comment:
Default is `UNKNOWN`

Reply:
solved in https://github.com/pagopa/io-functions-elt/pull/81/commits/edd3bb464fa3c5a9b3ad13efd0b6a5089e5b61a4